awx/main/models/events.py

```diff
@@ -570,6 +570,12 @@ def create_from_data(self, **kwargs):
 
         return self.objects.create(**kwargs)
 
+    def get_event_display(self):
+        '''
+        Needed for __unicode__
+        '''
+        return self.event
+
```

Comment:
@ryanpetrello I was getting this just spelunking around in the console

```
In [3]: iu.get_event_queryset()
Out[3]: ---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/venv/awx/lib/python2.7/site-packages/IPython/core/formatters.pyc in __call__(self, obj)
    670                 type_pprinters=self.type_printers,
    671                 deferred_pprinters=self.deferred_printers)
--> 672             printer.pretty(obj)
    673             printer.flush()
    674             return stream.getvalue()

/venv/awx/lib/python2.7/site-packages/IPython/lib/pretty.pyc in pretty(self, obj)
    381                             if callable(meth):
    382                                 return meth(obj, self, cycle)
--> 383             return _default_pprint(obj, self, cycle)
    384         finally:
    385             self.end_group()

/venv/awx/lib/python2.7/site-packages/IPython/lib/pretty.pyc in _default_pprint(obj, p, cycle)
    501     if _safe_getattr(klass, '__repr__', None) not in _baseclass_reprs:
    502         # A user-provided repr. Find newlines and replace them with p.break_()
--> 503         _repr_pprint(obj, p, cycle)
    504         return
    505     p.begin_group(1, '<')

/venv/awx/lib/python2.7/site-packages/IPython/lib/pretty.pyc in _repr_pprint(obj, p, cycle)
    699     """A pprint that just redirects to the normal repr function."""
    700     # Find newlines and replace them with p.break_()
--> 701     output = repr(obj)
    702     for idx,output_line in enumerate(output.splitlines()):
    703         if idx:

/venv/awx/lib/python2.7/site-packages/django/db/models/query.pyc in __repr__(self)
    227         if len(data) > REPR_OUTPUT_SIZE:
    228             data[-1] = "...(remaining elements truncated)..."
--> 229         return '<%s %r>' % (self.__class__.__name__, data)
    230 
    231     def __len__(self):

/venv/awx/lib/python2.7/site-packages/django/db/models/base.pyc in __repr__(self)
    588     def __repr__(self):
    589         try:
--> 590             u = six.text_type(self)
    591         except (UnicodeEncodeError, UnicodeDecodeError):
    592             u = '[Bad Unicode data]'

/awx_devel/awx/main/models/events.pyc in __unicode__(self)
    547 
    548     def __unicode__(self):
--> 549         return u'%s @ %s' % (self.get_event_display(), self.created.isoformat())
    550 
    551     @classmethod

AttributeError: 'InventoryUpdateEvent' object has no attribute 'get_event_display'
```

Reply:
Interesting.  This must be some sort of fallout from my job event model refactoring a few weeks back.  Good catch.